### PR TITLE
Added new functionality to file 2

### PR DESCRIPTION
**Pull request by cdeiderich**
- Added spinner when loading
- Fix bug for image display

### CODE REVIEW

**Comment by zainkhan305** [comment]
really like those changes you made. Everything looks good on my end. APPROVED

File: file2.html [hunk 1]
```diff
@@ -8,12 +8,11 @@ <h1>File 2</h1>
...
         <p>Cras molestie risus a enim convallis vitae luctus libero lacinia. Donec et nisi dictum felis sollicitudin congue. Nunc iaculis risus vel orci ornare dignissim sed vitae nulla. Donec a congue leo. Suspendisse <em>potenti</em>. </p>
         <p>Nulla auctor eleifend 23<sup>rd</sup> of May turpis consequat pharetra. Donec tempus tempus tellus, ac <abbr title='Hypertext Markup Language'>HTML</abbr> lacinia turpis mattis ac. Donec tempus tempus tellus, ac lacinia turpis mattis ac. Aliquam tincidunt velit sit amet ante hendrerit tempus. Vestibulum <code>text-align : justify</code> lacus. </p>
-        <p>Nulla auctor eleifend turpis consequat pharetra. Maecenas sit amet tellus nec mi gravida posuere non pretium magna. Vestibulum sit amet ipsum lacus. Sed dapibus, lectus sit amet adipiscing egestas, mauris est viverra nibh, <a href='#'>iaculis pretium</a> sem orci aliquet mauris. </p>
```
Comment (cdyths) [inline file2.html:15]: WTH is "eleifend", elephant?

**Comment by Jatavinr** [comment]
It needs more numbers

**Review by zainkhan305 — APPROVED** [review]

**Comment by pjr1985** [comment]
Pretend I'm saying something smart here.

File: file2.html [hunk 2]
```diff
@@ -8,12 +8,11 @@ <h1>File 2</h1>
...
+        <p>Nulla auctor ftert turpis consequat pharetra. Maecenas sit amet tellus nec mi gravida posuere non pretium magna. Vestibulum sit amet ipsum lacus. Sed dapibus, lectus sit amet adipiscing egestas, mauris est viverra nibh, <a href='#'>iaculis pretium</a> sem orci aliquet mauris. </p>
```
Comment (cdeiderich) [inline file2.html:14]: this change sucks

**Pull request merged**